landoui/assets_src/js/pages/RevisionPage.js

```diff
@@ -0,0 +1,29 @@
+$.fn.landoRevisionPage = function(){
+  function RevisionPage(){
+    let $page = $(this);
+    let state = $page.data('state');
+    $page.find('.RevisionPage-collapseLink').landoRevisionPageCollapsible();
+  }
+
+  return this.each(RevisionPage);
```

Comment:
`RevisionPage` is so small we can probably just inline it.  Also, only classes should be capitalized.